stable/prometheus-rabbitmq-exporter/README.md

```diff
@@ -65,6 +65,8 @@ The following table lists the configurable parameters and their default values.
 | `rabbitmq.timeout`       | timeout in seconds for retrieving data from management plugin          | `30`                      |
 | `rabbitmq.max_queues`    | max number of queues before we drop metrics (disabled if set to 0)     | `0`                       |
 | `annotations`            | pod annotations for easier discovery                                   | {}                        |
+| `prometheus.monitor.enabled` | Set this to `true` to create ServiceMonitor for Prometheus operator | `false` | |
+| `prometheus.monitor.additionalLabels` | Additional labels that can be used so ServiceMonitor will be discovered by Prometheus | `{}` | |
```

Comment:
A `prometheus.monitor.interval` option would be nice

Reply:
Added interval to servicemonitor